src/app/pages/connect-page/connection-details/connection-details.page.ts

```diff
@@ -29,7 +31,19 @@ export class ConnectionDetailsPage implements OnInit{
     //       self.navigateTo('/list-connections');
     //     }
     // }
-    
+    onRemoveBtnClick() {
+        let self = this;
+    this._alertService.show({
```

Comment:
Use proper indenting. Look at other code/classes to see how it's done in other classes.